pkg/router/template/router.go

```diff
@@ -29,16 +29,18 @@ const (
 	ProtocolHTTP  = "http"
 	ProtocolHTTPS = "https"
 	ProtocolTLS   = "tls"
-)
 
-const (
 	routeFile       = "routes.json"
 	certDir         = "certs"
 	caCertDir       = "cacerts"
 	defaultCertName = "default"
 
 	caCertPostfix   = "_ca"
 	destCertPostfix = "_pod"
+
+	// '-' is not used because namespace can contain dashes
+	// '_' is not used as this could be part of the name in the future
```

Comment:
Should we say why / can't be used?  The comment below implies that / can't be used because it needs to be written to disk.

Reply:
ok, I will mention about '/'